### DISCUSSION TITLE
Material Overide list for selected objects

### DISCUSSION THREAD

**Discussion by negicless** [bnpr/Malt#490] [Feedback & Feature Requests]
Is it possible to have a feature to set material overide for different collections/object, even better to have a list to add or remove objects from overide list.

**Comment by pragma37** [comment]
>  even better to have a list to add or remove objects from overide list.

You can change the default material in the world settings. Then every object that doesn't have a working material assigned will use the default material instead.

If that's not enough, I think it should be possible to override a collection material using a custom Render node.
